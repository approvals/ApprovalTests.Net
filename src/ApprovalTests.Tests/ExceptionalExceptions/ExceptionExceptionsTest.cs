using System;
using ApprovalTests.ExceptionalExceptions;
using NUnit.Framework;

namespace ApprovalTests.Tests.ExceptionalExceptions
{
    [TestFixture]
    public class ExceptionExceptionsTest
    {
        [Test]
        public void TestUniqueId()
        {
            var exceptionalId = Exceptional.GenerateUniqueId<Exception>();
            Approvals.Verify(exceptionalId);
        }


        //[Test]
        public void TestTlDr()
        {
            Approvals.Verify(new ExceptionalTlDr(new ExceptionalId
            {
                Assembly = "Assembly",
                Class = "Class",
                Exception = "Exception",
                Method = "Method",
            }));
        }
    }
}