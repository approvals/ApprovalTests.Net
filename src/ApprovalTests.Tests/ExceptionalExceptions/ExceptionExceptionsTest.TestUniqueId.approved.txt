ExceptionalId
{
	Assembly: ApprovalTests.Tests, Version=0.0.0.0, Culture=neutral, PublicKeyToken=11bd7d124fc62e0f
	Class: ApprovalTests.Tests.ExceptionalExceptions.ExceptionExceptionsTest
	Method: ExceptionExceptionsTest.TestUniqueId()
	Exception: System.Exception
}
